README.md

```diff
@@ -116,23 +115,23 @@ annotation such as `string[]` or `as any`.
 
 ```typescript
 // TypeScript
-import WalletLink from 'walletlink'
+import CoinbaseWallet from 'wallet-sdk'
 import Web3 from 'web3'
 
 const APP_NAME = 'My Awesome App'
 const APP_LOGO_URL = 'https://example.com/logo.png'
 const DEFAULT_ETH_JSONRPC_URL = 'https://mainnet.infura.io/v3/<YOUR_INFURA_API_KEY>'
 const DEFAULT_CHAIN_ID = 1
 
-// Initialize WalletLink
-export const walletLink = new WalletLink({
+// Initialize Coinbase Wallet SDK
+export const coinbaseWallet = new CoinbaseWallet({
```

Comment:
```suggestion
export const coinbaseWallet = new CoinbaseWalletSDK({
```

Reply:
what's the convention for the var name @bangtoven ?

Reply:
it doesn't matter much, but on the sdk repo we named it coinbaseWalletSDK
![image](https://user-images.githubusercontent.com/966662/156068713-ef3da5cd-f8c8-4b0d-8816-cc7925e237c8.png)